templates/experiments/components/experiment_actions_column.html

```diff
@@ -1,4 +1,5 @@
 {% if record.is_editable %}
+  {% include "chatbots/chatbot_copy_dialog.html"%}
```

Comment:
you can pass values into the include:
```suggestion
  {% include "chatbots/chatbot_copy_dialog.html" with chatbot_id=record.id team_slug=record.team.slug %}
```